packages/swapper/src/swappers/thorchain/utils/makeSwapMemo.test.ts

```diff
@@ -0,0 +1,33 @@
+import { makeSwapMemo } from './makeSwapMemo'
+
+describe('makeSwapMemo', () => {
+  it('make a trade to usdc memo with', () => {
```

Comment:
nitpick that test descriptions start with `should` and this one isn't a complete sentence